test/mssql-config.js

```diff
@@ -0,0 +1,9 @@
+let config = null
+
+try {
+  config = require(`./.mssql.json`)
+} catch (e) {
+  config = require(`./.mssql-docker.json`)
```

Comment:
what would be the expected error here?

Reply:
file doesn't exist?

Reply:
🤷‍♂️

Reply:
.mssql-docker.json docker file is included in repo with correct settings for dockerized mssql and if someone likes to use he's own mssql installation he can just add that .mssql.json which is not added to repo to override usage of docker... I don't have much of preference how it is signaled which configuration needs to be used. This was the easiest choice to implement and  maintain that I came up :) Arguable maybe not the most elegant, but I like promoting functional design over elegancy.

Reply:
I'm really not bothered, the only thing I *may* have changed is to call it `.mssql,dist.json`

Reply:
The configuration file is tightly coupled with the configuration in docker file, that is why I choose the name also to refer to docker. Though `.dist.json` suffix could be good in either way 👍